packages/cli/src/commands/testCommand.ts

```diff
@@ -63,6 +63,13 @@ export default async (cmd: Command) => {
     (process.env as any).NODE_ENV = 'test';
   }
 
+  // This is to have a consistent timezone for when running tests that involve checking
+  // the formatting of date/times.
+  // https://stackoverflow.com/questions/56261381/how-do-i-set-a-timezone-in-my-jest-config
+  if (!process.env.TZ) {
```

Comment:
You'll also need a changeset for this addition in the `cli` too :)

Reply:
Thanks @benjdlambert :) I've added it as a patch. I was thinking possible it should be a minor as it might affect tests outside the project if used externally, but I settled on a patch. Let me know if you think otherwise.